evaluation/e2e_samples/chunking/README.md

```diff
@@ -0,0 +1,95 @@
+# Chunking Evaluation <!-- omit in toc -->
```

Comment:
Are these Evaluation Metrics (In-top and RougeL) dependent on a particular chunking strategy? or could they be implemented here assuming that each "chunk" represented a unique text/document?

Reply:
No, these evaluation metrics are not dependent on a particular chunking strategy. They can be implemented assuming that each “chunk” represents a unique text/document. I just added a note on the document